src/Processor/LatexToUnicodeProcessor.php

```diff
@@ -63,12 +74,27 @@ private function decode($text)
                 $this->pandoc = new Pandoc();
             }
 
-            return $this->pandoc->runWith($text, [
-                'from' => 'latex',
-                'to' => 'plain',
-                'wrap' => 'none',
-            ]);
-        } catch (PandocException $exception) {
+            if ($this->isUeberdosisPandocAvailable()) {
+                // use ueberdosis/pandoc
+                $output = $this->pandoc->input($text)->execute([
+                    '--from', 'latex',
+                    '--to', 'plain',
+                    '--wrap', 'none',
+                ]);
+
+                // remove newline character added by Pandoc conversion
+                $output = substr($output, 0, -1);
+            } else {
+                // use ryakad/pandoc-php
+                $output = $this->pandoc->runWith($text, [
+                    'from' => 'latex',
+                    'to' => 'plain',
+                    'wrap' => 'none',
+                ]);
+            }
```

Comment:
Some suggestions:

- Checking if a package is installed or not should be done once, during the converter creation, not every time this processor is called
- We could wrap pandoc initialisation inside a callable
  - signature: `function (string $latex): string`
  - then we can rename `->pandoc` attribute to `->converter`
- We need tests with both libraries, we may need:
  - remove `ryakad/pandoc-php` as dev dependency
  - add them dynamically when running the CI
  - throw proper exception when none is installed